src/splat/segtypes/common/bin.py

```diff
@@ -33,3 +33,10 @@ def split(self, rom_bytes):
 
             f.write(rom_bytes[self.rom_start : self.rom_end])
         self.log(f"Wrote {self.name} to {path}")
+
+    @property
+    def statistics_type(self) -> SegmentType:
+        stats_type = self.type
+        if self.is_name_default():
+            stats_type = "unk"
```

Comment:
The old code was checking for `bin` types too.
I'm not sure if we should keep that check tho, naming the section itself could be considered as it being known, idk. 
What do you think @ethteck ?

Reply:
yeah I think considering it known if it's named is totally fine, personally